cime_config/tests.py

```diff
@@ -488,6 +488,19 @@
             )
     },
 
+    "e3sm_scream_v1_mpassi" : {
+        "time"  : "01:00:00",
+        "tests" : (
+            "ERP_D_Ln9.ne4_oQU240.F2010-SCREAMv1-MPASSI.atmlndactive-rtm_off",
+            "SMS_D_Ln9.ne4_oQU240.F2010-SCREAMv1-MPASSI-noAero",
+            "ERP_Ln22.ne4pg2_oQU480.F2010-SCREAMv1-MPASSI.atmlndactive-rtm_off",
+            "ERS_D_Ln22.ne4pg2_oQU480.F2010-SCREAMv1-MPASSI.atmlndactive-rtm_off",
+            "ERS_Ln22.ne30_oECv3.F2010-SCREAMv1-MPASSI.atmlndactive-rtm_off",
+            "PEM_Ln90.ne30pg2_EC30to60E2r2.F2010-SCREAMv1-MPASSI",
+            "ERS_Ln22.ne30pg2_EC30to60E2r2.F2010-SCREAMv1-MPASSI.atmlndactive-rtm_off",
```

Comment:
That's a lot of cases to add to our nightlies. Can we trim it down to 4-5 tests?

Reply:
The 3rd and 4th are very similar. We can run ERS manually as part of debugging, if the ERP one fails.

Reply:
Sounds good. Some of the tests were intended to replace those in lowres and medres suite (when the compset is adopted as default). For current purpose,  no more than three necessary (ERP, ERS_D and PEM).